.github/workflows/lint.yml

```diff
@@ -105,25 +105,11 @@ jobs:
           key: cargo-registry-${{ steps.get-month.outputs.month }}
           restore-keys: |
             cargo-registry-
-      - name: Install Rust packages
-        run: |
-          rustup component add rustfmt
-          # when updating these versions, also update the versions in "docs/developer_guide.md"
-          cargo install --force --version 0.24.3 cbindgen
-          cargo install --force --version 0.60.1 bindgen
       - name: Check bindings (release)
         run: |
-          rm -rf build && mkdir build
-          (cd build && cmake -DCMAKE_BUILD_TYPE=Release .. && make bindings)
+          (cd src && cargo build --release --features bindings)
```

Comment:
Interesting idea. I ended up adding it as a flag to the setup script's build command, and using that here. Maybe it'd make sense to make it a top-level setup script command instead (`setup bindings` instead of `setup build --bindings`), but I think I'm going to leave it for now.

In particular I wonder if it's worth revisiting just always regenerating now that I think we're fairly confident we're never going to switch over to manually editing (vs freshly regenerating), and with it in the build scripts it's a little less onerous (don't need to install the command-line tools)